ouroboros-consensus/src/Ouroboros/Consensus/Mempool/API.hs

```diff
@@ -1,13 +1,18 @@
+{-# LANGUAGE FlexibleContexts        #-}
+{-# LANGUAGE StandaloneDeriving      #-}
 {-# LANGUAGE TypeFamilies            #-}
+{-# LANGUAGE UndecidableInstances    #-}
```

Comment:
Mmm. Is this needed?

Reply:
Unfortunately, yes, for these instances:

```haskell
deriving instance Eq   (GenTx blk) => Eq   (TraceEventMempool blk)
deriving instance Show (GenTx blk) => Show (TraceEventMempool blk)
```